### ISSUE TITLE
Chatgml2-6b crashes during weight quantization in 254-llm-chatbot.ipynb

### ISSUE THREAD

**Issue by stokesj0001** [openvinotoolkit/openvino_notebooks#1471]
**Describe the bug**
Chatgml2-6b crashes during weight quantization.

 **Expected behavior**
I would expect the model to be quantized without crashing.

**Screenshots**
![image](https://github.com/openvinotoolkit/openvino_notebooks/assets/97129360/da209fea-38f7-4cbf-9475-5c2e44ee581b)

** Environment information **

I thought the issue might be that I have openvino installed alongside openvino-nightly. However I removed openvino-nightly and still have this issue.

The following are the packages in my environment:

about-time==4.2.1
absl-py==2.0.0
accelerate==0.24.1
addict==2.4.0
aiofiles==22.1.0
aiohttp==3.8.6
aiosignal==1.3.1
aiosqlite==0.19.0
alive-progress==3.1.5
altair==5.1.2
annotated-types==0.6.0
anyio==3.7.1
argon2-cffi==23.1.0
argon2-cffi-bindings==21.2.0
arrow==1.3.0
asttokens==2.4.1
astunparse==1.6.3
async-timeout==4.0.3
attrs==23.1.0
autograd==1.6.2
Babel==2.13.1
beautifulsoup4==4.12.2
bleach==6.1.0
cachetools==5.3.2
certifi==2023.7.22
cffi==1.16.0
charset-normalizer==3.3.2
click==8.1.7
cma==3.2.2
coloredlogs==15.0.1
comm==0.2.0
controlnet-aux==0.0.7
cycler==0.12.1
datasets==2.14.6
debugpy==1.8.0
decorator==5.1.1
defusedxml==0.7.1
Deprecated==1.2.14
diffusers==0.23.0
dill==0.3.7
einops==0.7.0
entrypoints==0.4
exceptiongroup==1.1.3
executing==2.0.1
fastapi==0.104.1
fastjsonschema==2.18.1
ffmpy==0.3.1
filelock==3.13.1
flatbuffers==23.5.26
fonttools==4.44.0
fqdn==1.5.1
frozenlist==1.4.0
fsspec==2023.10.0
future==0.18.3
gast==0.5.4
google-auth==2.23.4
google-auth-oauthlib==1.0.0
google-pasta==0.2.0
gradio==3.47.1
gradio_client==0.6.0
grapheme==0.6.0
grpcio==1.59.2
h11==0.14.0
h5py==3.10.0
httpcore==1.0.1
httpx==0.25.1
huggingface-hub==0.17.3
humanfriendly==10.0
idna==3.4
imageio==2.32.0
importlib-metadata==6.8.0
importlib-resources==6.1.1
ipykernel==6.26.0
ipython==8.17.2
ipython-genutils==0.2.0
ipywidgets==7.7.0
isoduration==20.11.0
jedi==0.19.1
Jinja2==3.1.2
joblib==1.3.2
json5==0.9.14
jsonpointer==2.4
jsonschema==4.19.2
jsonschema-specifications==2023.7.1
jstyleson==0.0.2
jupyter-events==0.9.0
jupyter-ydoc==0.2.5
jupyter_client==7.4.9
jupyter_core==5.5.0
jupyter_server==2.10.0
jupyter_server_fileid==0.9.0
jupyter_server_terminals==0.4.4
jupyter_server_ydoc==0.8.0
jupyterlab==3.6.3
jupyterlab-pygments==0.2.2
jupyterlab-widgets==1.1.0
jupyterlab_server==2.25.1
keras==2.14.0
kiwisolver==1.4.5
lazy_loader==0.3
libclang==16.0.6
Markdown==3.5.1
markdown-it-py==3.0.0
MarkupSafe==2.1.3
matplotlib==3.5.2
matplotlib-inline==0.1.6
mdurl==0.1.2
mistune==3.0.2
ml-dtypes==0.2.0
mpmath==1.3.0
multidict==6.0.4
multiprocess==0.70.15
natsort==8.4.0
nbclassic==1.0.0
nbclient==0.9.0
nbconvert==7.11.0
nbformat==5.9.2
nest-asyncio==1.5.8
networkx==2.8.2
ninja==1.10.2.4
nncf==2.7.0
notebook==6.5.6
notebook_shim==0.2.3
numpy==1.24.4
oauthlib==3.2.2
onnx==1.15.0
onnxruntime==1.16.2
opencv-python==4.8.1.78
openvino==2023.2.0
openvino-nightly==2023.3.0.dev20231115
openvino-telemetry==2023.2.1
opt-einsum==3.3.0
optimum==1.14.0
optimum-intel @ git+https://github.com/huggingface/optimum-intel.git@f248835b16ce4ec054d6d4d629dff4213fe94157
orjson==3.9.10
overrides==7.4.0
packaging==23.2
pandas==2.0.3
pandocfilters==1.5.0
parso==0.8.3
pexpect==4.8.0
Pillow==10.0.1
pipfile==0.0.2
platformdirs==3.11.0
prometheus-client==0.18.0
prompt-toolkit==3.0.39
protobuf==4.25.0
psutil==5.9.6
ptyprocess==0.7.0
pure-eval==0.2.2
pyarrow==14.0.1
pyasn1==0.5.0
pyasn1-modules==0.3.0
pycparser==2.21
pydantic==2.4.2
pydantic_core==2.10.1
pydot==1.4.2
pydub==0.25.1
Pygments==2.16.1
pymoo==0.6.0.1
pyparsing==2.4.7
python-dateutil==2.8.2
python-json-logger==2.0.7
python-multipart==0.0.6
pytz==2023.3.post1
PyYAML==6.0.1
pyzmq==24.0.1
referencing==0.30.2
regex==2023.10.3
requests==2.31.0
requests-oauthlib==1.3.1
rfc3339-validator==0.1.4
rfc3986-validator==0.1.1
rich==13.6.0
rpds-py==0.12.0
rsa==4.9
safetensors==0.4.0
scikit-image==0.22.0
scikit-learn==1.3.2
scipy==1.10.1
semantic-version==2.10.0
Send2Trash==1.8.2
sentencepiece==0.1.99
six==1.16.0
sniffio==1.3.0
soupsieve==2.5
stack-data==0.6.3
starlette==0.27.0
sympy==1.12
tensorboard==2.14.1
tensorboard-data-server==0.7.2
tensorflow==2.14.0
tensorflow-estimator==2.14.0
tensorflow-io-gcs-filesystem==0.34.0
termcolor==2.3.0
terminado==0.17.1
texttable==1.7.0
threadpoolctl==3.2.0
tifffile==2023.9.26
tiktoken==0.5.1
timm==0.9.10
tinycss2==1.2.1
tokenizers==0.14.1
toml==0.10.2
tomli==2.0.1
toolz==0.12.0
torch==2.1.0+cpu
torchaudio==2.1.0+cpu
torchvision==0.16.0+cpu
tornado==6.3.3
tqdm==4.66.1
traitlets==5.13.0
transformers==4.34.1
transformers-stream-generator==0.0.4
types-python-dateutil==2.8.19.14
typing_extensions==4.8.0
tzdata==2023.3
uri-template==1.3.0
urllib3==1.26.15
uvicorn==0.24.0.post1
wcwidth==0.2.9
webcolors==1.13
webencodings==0.5.1
websocket-client==1.6.4
websockets==11.0.3
Werkzeug==3.0.1
widgetsnbextension==3.6.0
wrapt==1.14.1
xxhash==3.4.1
y-py==0.6.2
yarl==1.9.2
ypy-websocket==0.8.4
zipp==3.17.0

**Comment by eaidova** [comment]
@stokesj0001 hello, looks like chatglm2 int4 weights compression is not a part of nncf release package,  we switched our notebook on installing nncf from github with version where fix already present.
Could you please check updated notebook version https://github.com/openvinotoolkit/openvino_notebooks/pull/1473?

**Comment by stokesj0001** [comment]
As of commit #1475 this issue appears resolved.
